disnake/guild.py

```diff
@@ -3227,6 +3234,9 @@ async def create_custom_emoji(
             "2", "150"
             "3", "250"
 
+        Emojis with :attr:`subscription roles <RoleTags.integration_id>` are considered premium emoji,
```

Comment:
This is missing a note saying that you can't mix subscription roles and non-subscription roles when creating a custom emoji, (here and Role.edit)

Reply:
clarified it a bit and added more details in https://github.com/DisnakeDev/disnake/pull/904/commits/07bf0b7569d07547982f0cfa7748bd83a77ecc95